quinn/src/send_stream.rs

```diff
@@ -245,6 +252,32 @@ impl SendStream {
     }
 }
 
+/// Check if a send stream is stopped.
+///
+/// Returns `Some` if the stream is stopped or the connection is closed.
+/// Returns `None` if the stream is not stopped.
+fn send_stream_stopped(
+    conn: &mut State,
+    stream: StreamId,
+    is_0rtt: bool,
+) -> Option<Result<Option<VarInt>, StoppedError>> {
+    if is_0rtt && conn.check_0rtt().is_err() {
+        Some(Err(StoppedError::ZeroRttRejected))
```

Comment:
Nit: use an early `return` for this so we can reduce rightward drift for the other branch.

Reply:
I can do, but I actually just changed it to that form 30min ago because I preferred the if/else branching to be visible. But your call ofc.

Reply:
Yes, I think we generally prefer early returns.